### PR TITLE
[Feature]: Added Login/SIgnup Screen (Only UI)

### PR DESCRIPTION
**Pull request by munjalSaksham71**
Desc: Added Only UI for Login and Signup Pages
Screenshot 2022-03-22 at 9 56 53 PM
Screenshot 2022-03-22 at 9 57 10 PM

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
looks good. Good to merge

**Review by JantuDeb — APPROVED** [review]
This looks good , I would suggest you to add the states for login and sign-up for future use.

File: src/App.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import ProductsScreen from "./screens/Products";
 import HomeScreen from "./screens/HomeScreen";
 import CartScreen from "./screens/CartScreen";
 import WishlistScreen from "./screens/WishlistScreen";
+import LoginScreen from "./screens/LoginScreen";
+import SignupScreen from "./screens/SignupScreen";
```
Comment (t007rushi) [inline src/App.js]: Make one ```index.js``` file export from there for single import

File: src/components/Nav/Navbar.jsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const Navbar = () => {
           <Link to="/cart" className="header_link link ">
           <BsCartFill /> Cart
           </Link>
-          <Link to="/" className="header_link link ">
+          <Link to="/login" className="header_link link ">
```
Comment (t007rushi) [inline src/components/Nav/Navbar.jsx]: Use Navlink instead of Link

File: src/screens/SignupScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom"
+import Navbar from "../components/Nav/Navbar"
+import './LoginScreen.css'
+
+const SignupScreen = () => {
+    return (
+        <div>
+        <Navbar />
+        <form className="wrap form-height">
+        <div className="heading1 mt-3">SIGN IN</div>
```
Comment (t007rushi) [inline src/screens/SignupScreen.jsx]: Change this to ```SIGN UP```

**Review by t007rushi — COMMENTED** [review]
Suggested some changes to refactor your code

**Comment by munjalSaksham71** [comment]
> This looks good , I would suggest you to add the states for login and sign-up for future use.

Thanks,  Will Create another PR for functionality- This one is just for UI.

**Review by t007rushi — APPROVED** [review]

**Pull request merged**